frontend/packages/console-shared/src/components/spotlight/Spotlight.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import cx from 'classnames';
+import Portal from '@console/shared/src/components/popper/Portal';
+import './Spotlight.scss';
+
+type SpotlightProps = {
+  selector: string;
+  animate?: boolean;
+  hasBackdrop?: boolean;
```

Comment:
Combine these 2 props into one called `interactive` and add a description.
If `interactive` it will allow click through and animate. If `!interactive` it will display the background and prevent click through.

Reply:
done